webcompat/form.py

```diff
@@ -48,32 +52,42 @@
 username_message = u'A valid username must be {0} characters long'.format(
     random.randrange(0, 99))
 
+desc_label = 'Please describe what was wrong <span class="wc-Form-required">' \
+    '*</span>'
+desc_message = u'An issue description is required.'
+
 problem_label = (u'What seems to be the trouble?',
                  '<span class="wc-Form-required">*</span>')
 url_label = u'Site URL <span class="wc-Form-required">*</span>'
 
-desc_default = u'''1. Navigate to: Site URL
-2. …
+steps_default = u'''1. I've tried to log in.
+2. I've filled out the form details.
+3. I clicked on the submit button.
+4. Nothing happened.
 
-Expected Behavior:
-
-Actual Behavior:
 '''
 
 
 class IssueForm(FlaskForm):
     '''Define form fields and validation for our bug reporting form.'''
     url = StringField(url_label,
                       [InputRequired(message=url_message)])
-    browser = StringField(u'Browser / Version', [Optional()])
+    browser = StringField(u'Is this information correct?', [Optional()])
     os = StringField(u'Operating System', [Optional()])
     username = StringField(u'Username',
                            [Length(max=0, message=username_message)])
-    description = TextAreaField(u'Give more details', [Optional()],
-                                default=desc_default)
+    description = StringField(desc_label,
+                              [InputRequired(message=desc_message)])
+
+    steps_reproduce = TextAreaField(u'How did you get there?', [Optional()],
+                                    default=steps_default)
     problem_category = RadioField(problem_label,
                                   [InputRequired(message=radio_message)],
                                   choices=problem_choices)
+    browser_test = RadioField(problem_label,
```

Comment:
Oh! check that. We are not using the problem_label in the form.html. It's written there instead of passing the variable. This is dead code. 
https://github.com/webcompat/webcompat.com/blob/master/webcompat/templates/home-page/form.html#L17-L31

This was added by @miketaylr two years ago.
https://github.com/webcompat/webcompat.com/search?q=problem_label&type=Commits&utf8=%E2%9C%93

but was never used.
https://github.com/webcompat/webcompat.com/blob/ec418f78ba0209a9c06f04fdd089aa4af30ebb8a/webcompat/templates/form.html#L7


No wonder you are confused.

So we can remove `problem_label` safely from the source code of `form.py`

a good way to test a library is to use the console.

```python
>>> from wtforms import Form, RadioField
>>> from wtforms.validators import Optional
>>> browser_test_choices = [(u'yes', u'Yes'), (u'no', u'No')]
>>> problem_label = (u'What seems to be the trouble?', '<span class="wc-Form-required">*</span>')
>>> class MyForm(Form):
...     browser_test = RadioField(problem_label, [Optional()], choices=browser_test_choices)
... 
>>> form = MyForm()
>>> str(form.browser_test)
'<ul id="browser_test"><li><input id="browser_test-0" name="browser_test" type="radio" value="yes"> <label for="browser_test-0">Yes</label></li><li><input id="browser_test-1" name="browser_test" type="radio" value="no"> <label for="browser_test-1">No</label></li></ul>'
```